trino/dbapi.py

```diff
@@ -302,6 +302,21 @@ def setinputsizes(self, sizes):
     def setoutputsize(self, size, column):
         raise trino.exceptions.NotSupportedError
 
+    def _remove_trailing_semicolon(self, statement: str) -> str:
+        """
+        Remove a trailing semicolon and whitespace from the sql statement if present.
+
+        :param statement: sql statement
+        :return: modified sql statement
+        """
+        remove_ws = statement.rstrip()
```

Comment:
Add early return if there is no semicolon in the statement.

Reply:
Exit's early if no semicolon is present in the statement.